util/config/gather-cray-prgenv-arguments

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+
+THIS_DIR=`dirname "$0"`
```

Comment:
This will evaluate to the empty string in cases where $0 is found on the PATH. This may seem unlikely, but it can happen if `.` is on the PATH and the script is run from this dir. A more bullet proof solution is:

```bash
THIS_DIR=$(cd $(dirname $0) ; pwd)
```